contracts/zero-ex/contracts/src/features/UniswapFeature.sol

```diff
@@ -39,12 +39,8 @@ contract UniswapFeature is
     string public constant override FEATURE_NAME = "UniswapFeature";
```

Comment:
Just wanted to confirm that integrators are no longer using the legacy proxy?

Reply:
API will stop supporting it march 31